CAIPs/caip-x.md

```diff
@@ -0,0 +1,99 @@
+
+### Chain Agnostic Improvement Proposal (CAIP) for Wallet-to-DApp Communication Standard
+
+  
+
+#### Abstract
+
+This CAIP aims to standardize the method by which extension based wallets can announce their canonical extension ID to decentralized applications (dApps). The goal is to enable dApps to communicate with wallets using the `externally_connectable` API in browsers, facilitating a secure and standardized interaction pattern across different blockchain platforms. This proposal builds upon EIP-6963, adapting its framework to the unique requirements of browser extension-based wallets and the external communication APIs provided by browsers.
+
+### Motivation
+
+First specifically why switch from an injected provider to `externally_connectable`
+
+#### Enhanced Security Benefits
+
+ **Reduced Permissions Requirement**: Utilizing the `externally_connectable` API allows wallet extensions communicate with websites, without requiring broad permissions to read and modify all data on the websites a user visits as is currently the case with the typical injected provider used by most extension Wallets. For instance the `externally_connectable` method does not allow modification of web page content. This reduces the level of trust and permission users must grant to the extension, minimizing potential security risks if the extension or its updates are compromised.
+    
+**Reduced Fingerprinting Risk**: The traditional method of injecting a JavaScript provider into every webpage a user visits exposes the user to increased fingerprinting risks. Each script injection can potentially be detected by malicious websites aiming to track users by identifying unique behaviors in their browser environments.
+
+#### Enhanced Performance Benefits
+
+**Fewer Blocking Scripts**: Injected scripts often modify the DOM or execute operations that can block or slow down page loading, especially if they are not efficiently optimized. By using the `externally_connectable` approach, extensions do not inject any JavaScript into the pages, thereby avoiding the introduction of blocking scripts that can degrade the performance of web pages.
+    
+ **Optimized Resource Usage**: The `externally_connectable` method focuses on direct communication through message passing, which is generally more resource-efficient than script injection. This means less CPU and memory usage as the communication is handled by the browser’s built-in mechanisms rather than through additional JavaScript execution on the page.
+
+**Faster Page Load Times**: Without the overhead of injecting and running additional scripts on every page, pages load faster, providing a smoother and more responsive user experience. This performance improvement is critical for user retention and satisfaction, especially on resource-intensive applications or slower devices.
```

Comment:
our CAIPs definitely overlap on this. I wonder if one caip can reference the other